src/assets/wise5/services/cRaterService.ts

```diff
@@ -301,4 +303,49 @@ export class CRaterService {
         return isAvailable;
       });
   }
+
+  public getDataFromResponse(response: any): any {
+    const data: any = {};
```

Comment:
Can we change this function to return type CRaterResponse instead of any?
```suggestion
  public getDataFromResponse(response: any): CRaterResponse {
    const data: CRaterResponse = new CRaterResponse();
```